### ISSUE TITLE
Unable to connect to redis db

### ISSUE THREAD

**Issue by k33g** [flywheelsports/fwsp-hydra#39]
Hi 👋 

I would like to connect to the redis db with something like that:
```json
    "redis": {
      "url": "redis://:oOLzF44DnFLzcZ5fsMZ@cktrezr8w-redis.services.hola.com:3002", // fake url
      "db":15
    }
```

If i use directly node redis client, it's ok:

```javascript
var redis = require("redis"),
    client = redis.createClient({
      url:"redis://:oOLzF44DnFLzcZ5fsMZ@cktrezr8w-redis.services.hola.com:3002"
    });

client.set("yo", "OK");

// This will return a JavaScript String
client.get("yo", function (err, reply) {
    console.log(reply.toString());
});
```

But with Hydra, I get some errors:

```shell
ERROR
{ event: 'error',
  message: 'Established Redis server connection has closed' }
ERROR
{ event: 'error', message: 'Reconnecting to redis server...' }
ERROR
{ event: 'error', message: 'Reconnecting to redis server...' }
ERROR
{ event: 'error', message: 'Reconnecting to redis server...' }
ERROR
{ event: 'error', message: 'Reconnecting to redis server...' }
ERROR
{ event: 'error',
  message: 'Max redis connection retry period exceeded.' }
ERROR
{ event: 'error',
  message: 'Error: Unable to establish a connection to Redis' }
```

so I don't understand how to set correctly my connection to redis

Can you help me please? 🙏  😄

**Comment by emadum** [comment]
We fixed this in [0.14.0](https://github.com/flywheelsports/fwsp-hydra/pull/38). I just published it on npm, so try updating and seeing if it works for you now.

**Comment by cjus** [comment]
@k33g Let us know if the 0.14.2 addresses your need.
